src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/ResourceManipulationCmdletBase.cs

```diff
@@ -36,7 +36,7 @@ public abstract class ResourceManipulationCmdletBase : ResourceManagerCmdletBase
         /// <summary>
```

Comment:
@rnikhilkumar This is ainappropriate, you should not have a dependency on this in your cmdlets.  If you need access for your tests, add an InternalsVisibleTo attribute to assemblyinfo.cs